docs/specifications/tests/Nameserver-TP/nameserver13.md

```diff
@@ -0,0 +1,87 @@
+# nameserver13: Test for EDNS truncated response
+
+### test case identifier
+
+**Nameserver13**
+
+### objective
+
+EDNS is a mechanism to announce capabilities of a DNS implementation,
+and is now basically required by any new functionality in DNS such as
+DNSSEC ([rfc 6891]).
+
+[RFC 6891, section 7] states that an OPT record is supposed to be included
+in a truncated response, where-in in the request, an OPT pseudo-RR is present.
+
+In this test case, we will query with DO bit set to '1' and forcing a
+truncated response with setting buffer size to 512 in the query. We will also
+make sure that the resolution is not retried using TCP.
+
+### Inputs
+"Child Zone" - The domain name to be tested.
+
+### Ordered description of steps to be taken to execute the test case
+
+1. Create a query for the *Child Zone* with DO bit set to '1', a buffer size
+of 512 bytes and with the option for ignoring truncation of UDP responses.  
```

Comment:
There is no such option unless you would use dig.

Reply:
I have re-worded the statement. Please check.